lib/ansible/modules/windows/win_copy.py

```diff
@@ -91,6 +91,7 @@
       end with "/", the directory itself with all contents is copied.
     - If path is a file and dest ends with "\", the file is copied to the
       folder with the same filename.
+    - Not required if using C(content)
```

Comment:
This seems to contradict `required: yes`. If this is not always required, also change to `required: no` and note cases in which it is required.

Reply:
Nice catch @samdoran. @ShachafGoldstein if this parameter is not required in all cases, remove the `required: yes` line - we only add a line for `required` when `required: yes`.

Reply:
Maybe the new line 94 could be `- Required unless using C(content).`